pandas/io/formats/format.py

```diff
@@ -656,6 +656,10 @@ def _chk_truncate(self) -> None:
                 frame = concat(
                     (frame.iloc[:, :col_num], frame.iloc[:, -col_num:]), axis=1
                 )
+                # truncate formatter
+                if is_list_like(self.formatters) and self.formatters:
```

Comment:
Do you need the `and self.formatters` piece here? Wouldn't this always be short-circuited when list_like?

Reply:
When running tests, somehow is possible that self.formatters could be an empty list. In this case when trying to slice, a lot of tests failed.

Reply:
Not sure I follow - can you provide an example or call out the test(s) that fail?

Reply:
Actually, every truncation test that's doesn't provide the atribute formatters fails. Just as example : test_to_html_alignment_with_truncation is one of them.

Reply:
df = DataFrame(np.arange(64).reshape(8, 8), index=row_index, columns=column_index)
        result = df.to_html(
>           max_rows=4, max_cols=4, index=index, header=header, index_names=index_names
        )

Reply:
df = DataFrame(np.arange(64).reshape(8, 8), index=row_index, columns=column_index)
        result = df.to_html(max_rows=4, max_cols=4, index=index, header=header, index_names=index_names)